client/app/bundles/course/admin/pages/LessonPlanItemSettings/translations.intl.js

```diff
@@ -13,6 +13,10 @@ const translations = defineMessages({
     id: 'course.admin.lessonPlanSettings.lessonPlanItemSettings',
     defaultMessage: 'Lesson Plan Item Settings',
   },
+  visible: {
+    id: 'course.admin.LessonPlanItemSettings.visible',
+    defaultMessage: 'Visible?',
```

Comment:
Is 'display?' and 'visible? ambiguous? A bit verbose, but would something like 'Allow on Lesson Plan?" and "Visible by Default" be clearer?